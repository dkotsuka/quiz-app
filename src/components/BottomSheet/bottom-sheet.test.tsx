import React from 'react'
import {Text} from 'react-native'
import { BottomSheet } from '.'
import {create} from 'react-test-renderer';

jest.useFakeTimers();
describe("BottomSheet", () => {

    it("visible match snapshot", () => {
        const component = create(<BottomSheet isVisble><Text>My component</Text></BottomSheet>)
        expect(component).toMatchSnapshot()
    })

    it("invisible match snapshot", () => {
        const component = create(<BottomSheet isVisble={false}><Text>My component</Text></BottomSheet>)
        expect(component).toMatchSnapshot()
    })
})