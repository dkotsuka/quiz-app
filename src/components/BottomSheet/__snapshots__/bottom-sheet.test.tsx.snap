// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomSheet invisible match snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "bottom": -150,
      "elevation": 3,
      "left": 0,
      "paddingHorizontal": 42,
      "paddingVertical": 14,
      "position": "absolute",
      "right": 0,
      "shadowColor": "#000000",
      "shadowOpacity": 0.4,
      "shadowRadius": 3,
      "zIndex": 10,
    }
  }
>
  <Text>
    My component
  </Text>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
  />
</View>
`;

exports[`BottomSheet visible match snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "bottom": -150,
      "elevation": 3,
      "left": 0,
      "paddingHorizontal": 42,
      "paddingVertical": 14,
      "position": "absolute",
      "right": 0,
      "shadowColor": "#000000",
      "shadowOpacity": 0.4,
      "shadowRadius": 3,
      "zIndex": 10,
    }
  }
>
  <Text>
    My component
  </Text>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
  />
</View>
`;
